packages/libs/core/src/rewrite.ts

```diff
@@ -0,0 +1,66 @@
+import { compileDestination, matchPath } from "./match";
+import { RewriteData, RoutesManifest } from "./types";
+import { addDefaultLocaleToPath } from "./locale";
+
+/**
+ * Get the rewrite of the given path, if it exists. Otherwise return null.
+ * @param path
+ * @param routesManifest
+ * @param router
+ * @param normalisedPath
+ */
+export function getRewritePath(
+  uri: string,
+  routesManifest: RoutesManifest
+): string | null {
```

Comment:
Maybe better if we are consistent and return `undefined` always instead of `null`? I guess not in scope of this PR but something I noticed. See: https://palantir.github.io/tslint/rules/no-null-keyword/. Probably I can add an eslint rule for it later on.

Reply:
Yes, I agree and did make the same change to some other functions. I can do that here.